collections/_infographics/policies/fit-for-55.md

```diff
@@ -1,13 +1,134 @@
+
+The package is currently at the stage of legislative proposals by the European Commission, which are being further discussed in the European Parliament and the Council of the European Union, i.e. with the participation of ministers of the governments of the Member States and national working groups. In this context, the Czech Republic's **presidency of the Council of the EU**,[^presidency-eu], which it will assume in the second half of 2022, is crucial and will thus have the opportunity to influence the direction of the negotiations. It is expected that negotiations on the final form of the legislation could last until 2024.[^fit-for-55-q-a]
+
+## Context of origin
+
+'Fit for 55' is part of the European Union's (EU) broader strategy to achieve  **<glossary id="carbon-neutrality">Carbon-neutrality</glossary>** by 2050, known as the [**European Green Deal**](https://ec.europa.eu/info/strategy/priorities-2019-2024/european-green-deal).  The European Commission presented the Green Deal in 2019, after which member states agreed on it in 2020. Then, in the summer of 2021, the [**European Climate Law Framework**](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A32021R1119) came into force, making the achievement of carbon neutrality by 2050 legally binding. This law includes a mid-term target that commits the EU to a 55% reduction in greenhouse gas emissions by 2030 compared to 1990, which the 'Fit for 55' package (loosely translated as "ready for 55%") is designed to ensure is achieved.
+
+**To keep global warming below 1.5°C**, relative to pre-industrial levels, **we need to halve greenhouse gas emissions globally** (not just within the EU) by 2030. The European Union (EU-27, excluding the UK) accounted for 7.5% of these emissions in 2020. In absolute terms, EU countries produced 2.6 billion tonnes of CO2 emissions in the same year, ranking third behind China and the US [^emission-co2].
+
+Although the EU and the world are arguably facing the biggest economic and societal transformation in history, according to Eurobarometer data from 2021, the majority of EU citizens (93%) perceive climate change as a serious problem. In the Czech Republic, the figure is over 85%. At the same time, 90% of Europeans and 87% of Czechs agree with the statement, "We should reduce greenhouse gas emissions to a minimum while offsetting the remaining emissions, for example, by expanding forest cover to make the EU economy climate neutral by 2050."[^eurobarometer]
```

Comment:
- Change "At the same time," to "In addition,". "at the same time" is typically used to preface a statement of contradiction.

Reply:
Thanks! I updated the text.